### DISCUSSION TITLE
Some Questions

### DISCUSSION THREAD

**Discussion by Jay123210599** [staxrip/staxrip#1290] [Q&A] [answered]
1. How do I batch hardcode subtitles from multiple videos at once?
2. How do I make image sequences from videos?
3. How do I make gifs/apngs/animated webps from videos?
4. Copy/Mux only works for cutting videos only works at keyframes/I-frames, right?

**Comment by Dendraspis** [comment, marked as answer]
1. Currently the only way would be to user a filter and use macros to point it at the right destination, but it would also require them to be named identically or like the source file or so.
2. For example:  https://avisynthplus.readthedocs.io/en/latest/avisynthdoc/corefilters/imagewriter.html
3. `Tools » Animation » ...`
4. Right

**Reply by Jay123210599** [reply]
For 1, how? For 2, how would that help?

**Reply by Dendraspis** [reply]
1. For example using `%source_dir%` or `%source_name%`
2. I does what you have asked for.

**Reply by Jay123210599** [reply]
Does Staxrip already have ImageWriter?

Can you give me a step-by-step tutorial on how to batch hardcode subtitles or does one already exists?
